stage1/rootfs/usr_from_src/Makefile

```diff
@@ -13,6 +13,7 @@ install: Makefile host_deps.txt
 	echo cp -af ../usr_from_src/systemd_build/installed/. \"\$${ROOT}\" >> install.tmp
 	cat host_deps.txt | while read dep; do echo install -D \"$${dep}\" \"\$${ROOT}/$${dep}\"; done >> install.tmp
 	echo ln -sf src \"\$${ROOT}/flavor\" >> install.tmp
+	echo "echo $(RKT_STAGE1_SYSTEMD_VER) > \"\$${ROOT}/systemd-version\"" >> install.tmp
```

Comment:
I like that it's more resistant to a new line at the end of the file so it can be edited manually for tests more easily.